### PR TITLE
114 main home pages implementation: fix images 

### PR DESCRIPTION
**Pull request by JoMarianek**
1. Task: [link]
2. Screenshot: 
image

3. Change description: Updated image path
4. Change rationale

### CODE REVIEW

File: src/views/HomeView.vue [hunk 1]
```diff
@@ -12,7 +12,7 @@
                     url: "",
                     parentId: null,
                     ctId: null,
-                    image: "src/assets/img/cat4.jpg",
+                    image: new URL('@/assets/img/cat4.jpg', import.meta.url).href,
```
Comment (vetalapo) [inline src/views/HomeView.vue:15]: Could you please change single quotes ' for double quotes " ?

**Review by vetalapo — APPROVED** [review]

**Review by valeryiay — APPROVED** [review]
Good! Thank you

**Comment by valeryiay** [comment]
Screenshot 2024-06-15 215758
Could we include burger menu in this PR too?

**Comment by JoMarianek** [comment]
Thanks! I'd rather merge this PR now and open a separate branch for burger menu as I haven't implemented it yet.

**Pull request merged**